### ISSUE TITLE
A list of commonly tweaked parameters and how to tweak them in the experiment CLI

### ISSUE THREAD

**Issue by laserkelvin** [IntelLabs/matsciml#282]
It would be very handy to have a subsection in the [experiment CLI documentation](https://matsciml.readthedocs.io/en/latest/experiment-interface.html) that lists out things that are regularly tweaked, and which config file to modify. Some parameters that come to mind are:

- Learning rate (sits in model/task definition)
- Callbacks (trainer config)
- DDP training
- Data loader workers and batch size

It is redundant since some of the things are already covered in the full text, but would be nice to have a very short and snappy section that lets you know at a glance. Feel free to add to the list.

@lory-w if you have things to add as you discover them, you can omnibus them in if you beat @melo-gonzo 😸

**Comment by laserkelvin** [comment]
Closed by #295.
